lib/src/SofaGLFW/SofaGLFWImgui.cpp

```diff
@@ -75,7 +76,86 @@ void imguiInit()
     io.ConfigFlags |= ImGuiConfigFlags_ViewportsEnable;
 
     // Setup Dear ImGui style
-    ImGui::StyleColorsDark();
+    ImVec4* colors = ImGui::GetStyle().Colors;
```

Comment:
SofaGLFWImgui starts to be quite big (1.5K loc), could some stuff could be moved into a SofaImguiconfig struct or something like that ? maybe in other pr

Reply:
I agree, I wanted to split the file but I focused more on the features than the cleanliness

Reply:
👍
just lets not forget about it